Packs/CortexAttackSurfaceManagement/IncidentFields/incidentfield-ASM_-_Dev_Check.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I got a validation error on this file.  Can you please format with "demisto-sdk format -i <path>"

Reply:
this is taking forever (updates) -- I'll push when it's done.

Reply:
-nv for "no validate" is faster

Reply:
done